ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -516,6 +516,13 @@ import '../nuxeo-button-styles.js';
       };
     }
 
+    _isChecked(selectAllActive, _excludedItems) {
+      if (selectAllActive && _excludedItems.length === 0) {
+        return true;
+      }
+      return false;
+    }
```

Comment:
This could be made shorter:
```suggestion
    _isChecked(selectAllActive, _excludedItems) {
      return selectAllActive && _excludedItems.length === 0;
    }
```